src/mca/bfrops/base/base.h

```diff
@@ -158,47 +160,209 @@ PMIX_EXPORT extern pmix_bfrops_globals_t pmix_bfrops_globals;
 #error Unsupported pid_t size!
 #endif
 
+#define PMIX_BFROPS_FLEX_BASE7_MAX_BUF_SIZE (SIZEOF_SIZE_T+1)
```

Comment:
I think you need to provide some explanation in the form of comments here - it isn't clear what you are doing in these macros, and why you are doing it.

Reply:
Agree.
@karasevb should we put a message from my commit here?
I think it explains the idea for positive numbers.
So please append description for negative ones and this should be complete enough.

Reply:
Done.